src/getHumanAge.test.js

```diff
@@ -1,10 +1,31 @@
 'use strict';
 
-describe('getHumanAge', () => {
-  const { getHumanAge } = require('./getHumanAge');
+const { getHumanAge } = require('./getHumanAge');
 
+describe('getHumanAge', () => {
   test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+
+  test('should correctly convert cat and dog ages to human ages', () => {
```

Comment:
Please mind test design techniques and create separate test for each separate check